server/src/DBTablesLastInfo.cc

```diff
@@ -207,9 +207,8 @@ HatoholError DBTablesLastInfo::getLastInfoList(LastInfoDefList &lastInfoDefList,
 	getDBAgent().runTransaction(arg);
 
 	const ItemGroupList &grpList = arg.dataTable->getItemGroupList();
-	ItemGroupListConstIterator itemGrpItr = grpList.begin();
-	for (; itemGrpItr != grpList.end(); ++itemGrpItr) {
-		ItemGroupStream itemGroupStream(*itemGrpItr);
+	for (auto item : grpList) {
```

Comment:
I propose 'itemGrp' as a variable name, because it is not a single item but a set of items.